aws/resource_aws_s3_bucket_test.go

```diff
@@ -2509,7 +2507,7 @@ func testAccCheckAWSS3BucketCreateViaCloudFormation(n string, stackId *string) r
 		conn := testAccProvider.Meta().(*AWSClient).cfconn
 		stackName := acctest.RandomWithPrefix("tf-acc-test-s3tags")
 		templateBody := fmt.Sprintf(`
-        {
+		{
```

Comment:
This block would be better aligned to the left edge

Reply:
Fixed!